style/icons/shield40_us_ky_parkway.svg

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Please save as "optimized svg" so we don't commit all the extra Inkscape stuff into the repo.